drivers/sensor/lsm6dso/lsm6dso.c

```diff
@@ -766,28 +744,35 @@ static int lsm6dso_init_chip(const struct device *dev)
 
 	k_busy_wait(100);
 
-	if (lsm6dso_accel_set_fs_raw(dev,
-				     LSM6DSO_DEFAULT_ACCEL_FULLSCALE) < 0) {
-		LOG_DBG("failed to set accelerometer full-scale");
+	fs = cfg->accel_range;
+	LOG_DBG("accel range is %d", fs);
+	if (lsm6dso_accel_set_fs_raw(dev, fs) < 0) {
+		LOG_DBG("failed to set accelerometer full-scale %d", fs);
```

Comment:
This should be `LOG_ERR`. It's also a little confusing that you call it "accelerometer full-scale" here, but "accel range" two lines above.

Reply:
ACK to this one and the rest of the observations here below as well. Thanks!